src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -1019,13 +1019,13 @@ static void glb_shutdown_locked(grpc_lb_policy* pol) {
     grpc_call_cancel(lb_call, nullptr);
     /* lb_on_server_status_received will pick up the cancel and clean up */
   }
-  if (glb_policy->retry_timer_active) {
+  if (glb_policy->retry_timer_callback_pending) {
     grpc_timer_cancel(&glb_policy->lb_call_retry_timer);
-    glb_policy->retry_timer_active = false;
+    glb_policy->retry_timer_callback_pending = false;
```

Comment:
As I think we discussed in another PR, we should not be setting any of these values to false outside of the timer callback function itself.  Let's go ahead and make that change as part of this PR, so that the semantics are clear.

Reply:
Done.